### PR TITLE
Characters

### PR DESCRIPTION
**Pull request by PhilippAubert**
Setup the API, now loading. Add some CSS-styling.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,46 @@
...
+import Characters from "./components/Characters.js";
+import CharactersId from "./components/CharactersId.js";
+import Home from "./components/Home.js";
+import Footer from "./components/Footer.js";
 
 function App() {
-  return <div className="App">Rick & Morty</div>;
+  return (
+    <Router>
+      {" "}
```
Comment (ManuelRemer) [inline src/App.js:13]: Moin Philipp,
you can delete line 13. Or is it a placeholder or something like that?

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+        <header className="Header-Grid">
+          <div className="Top-Menu">
+            <Header />
+            <Nav />
+          </div>
+        </header>
+        <main className="Main-Grid">
+          <Switch>
+            <Route path="/characters/:id">
+              <CharactersId />
```
Comment (ManuelRemer) [inline src/App.js:24]: in line 23: is this colon needed, I am not sure. Are you using useParam? If not this wont work, i think. I am checking some resources. Let me now, if I am on wrong track.

Comment (ManuelRemer) [reply src/App.js:24]: I am on the wrong track.

File: src/components/Characters.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    return characters.map((character) => {
+      const { id, name, image } = character;
+      return (
+        <div key={id}>
+          <Link to={`/characters/${id}`}>
+            <div class="Character-Grid">
+              <figure>
+                <img className="Character-Image" src={image} alt={name} />
+                <figcaption className="Character-Text">{name}</figcaption>
+              </figure>
```
Comment (ManuelRemer) [inline src/components/Characters.js:39]: oh, that's new to me. Figure and figcaption. Cool. Inspiration.

File: src/components/Characters.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    if (page < allPages) {
+      setPage(page + 1);
+    }
+  }
+
+  function createCharacters() {
+    return characters.map((character) => {
+      const { id, name, image } = character;
+      return (
+        <div key={id}>
```
Comment (ManuelRemer) [inline src/components/Characters.js:33]: You could create a CharacterComponent, and use it in createCharacters

**Review by ManuelRemer — COMMENTED** [review]
Hey Philipp, looks good, so far. If it is working for you, you can explain me that colon thing in App.js, left a comment there

File: src/components/Characters.js (hunk 3 above)
Comment (PhilippAubert) [reply src/components/Characters.js:39]: These are merely workarounds. I used them to have the text and photo together - didn't really work though. The figure-thing was an just an idea.

File: src/components/Characters.js (hunk 4 above)
Comment (PhilippAubert) [reply src/components/Characters.js:33]: I know. I didn't dare yet ;)

File: src/App.js (hunk 2 above)
Comment (PhilippAubert) [reply src/App.js:24]: I tried to follow Namir's example. Didn't include them at first

**Comment by PhilippAubert** [comment]
Thank you for reviewing !

File: src/components/Characters.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.MoreButton {
+  padding: 2%;
+  background-color: orange;
+  border-radius: 4px 4px 4px 4px;
```
Comment (hugh-burgess) [inline src/components/Characters.css:4]: you can just say 4px on its own as that will cover all four sides :)

**Review by hugh-burgess — CHANGES_REQUESTED** [review]
LGTM!

File: src/components/Characters.css (hunk 5 above)
Comment (PhilippAubert) [reply src/components/Characters.css:4]: 👍

File: src/App.css [hunk 6]
```diff
@@ -1,8 +1,20 @@
...
 }
+
+.HeaderBox {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.Top-Menu {
```
Comment (FriederikeM) [inline src/App.css:16]: in line 9 you use camel case but here you use hyphenation, you want to be consistent with your naming

File: src/components/CharactersId.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+function CharactersId() {
+  return <div></div>;
+}
+
+export default CharactersId;
```
Comment (FriederikeM) [inline src/components/CharactersId.js:5]: Can't see you using this component anywhere, but I assume it will end up being the individual character page

**Review by FriederikeM — APPROVED** [review]
Looking good! Only left minor comments

File: src/components/CharactersId.js (hunk 7 above)
Comment (PhilippAubert) [reply src/components/CharactersId.js:5]: You're correct! I've actually been working on this file right now.

File: src/App.css (hunk 6 above)
Comment (PhilippAubert) [reply src/App.css:16]: Thanks for pointing out. I'll leave that for the finishing touches.

**Pull request merged**